storage/container_storage_interface/persistent-storage-csi-ibm-powervs-block.adoc

```diff
@@ -9,13 +9,16 @@ toc::[]
 [id="persistent-storage-csi-ibm-powervs-block-introduction_{context}"]
 == Introduction
 
-The {ibm-power-server-name} Block CSI Driver will be installed through {ibm-power-server-name} Block CSI Driver Operator and the operator is based on libarary-go. The OpenShift library-go is a collection of functions that allow us to build OpenShift operators easily. Most of the functionality of a CSI driver operator is already available there. The {ibm-power-server-name} Block CSI Driver Operator is installed by the cluster-storage-operator. The Cluster-storage-operator installs the {ibm-power-server-name} Block CSI Driver Operator if the Platform type is Power Virtual Servers.
+The {ibm-power-server-name} Block CSI Driver is installed through {ibm-power-server-name} Block CSI Driver Operator and the operator is based on `library-go`. The {product-title} `library-go` framework is a collection of functions that allows users to build OpenShift operators easily. Most of the functionality of a CSI driver operator is already available there. The {ibm-power-server-name} Block CSI Driver Operator is installed by the Cluster Storage Operator. The Cluster Storage Operator installs the {ibm-power-server-name} Block CSI Driver Operator if the platform type is Power Virtual Servers.
```

Comment:
```suggestion
The {ibm-power-server-name} Block CSI Driver is installed through the {ibm-power-server-name} Block CSI Driver Operator and the operator is based on `library-go`. The {product-title} `library-go` framework is a collection of functions that allows users to build OpenShift operators easily. Most of the functionality of a CSI Driver Operator is already available there. The {ibm-power-server-name} Block CSI Driver Operator is installed by the Cluster Storage Operator. The Cluster Storage Operator installs the {ibm-power-server-name} Block CSI Driver Operator if the platform type is Power Virtual Servers.
```

Reply:
Marking for consistency but since they were in the original text, I don't think it's crucial to fix them.